sql/org_mozilla_firefox_derived/clients_last_seen_v1/query.sql

```diff
@@ -18,32 +18,24 @@ WITH cls_yesterday AS (
     clients_last_seen_v1 AS cls
   WHERE
     submission_date = DATE_SUB(@submission_date, INTERVAL 1 DAY)
--- Filter out rows from yesterday that have now fallen outside the 28-day window.
-    AND udf.shift_28_bits_one_day(baseline.days_seen_bits) > 0
```

Comment:
This check moves to the end of the query so that we can filter out rows that only have metrics ping information.